test/e2e/specs/specs-playwright/wp-signup__free.ts

```diff
@@ -0,0 +1,137 @@
+/**
+ * @group calypso-release
+ */
+
+import {
+	DataHelper,
+	DomainSearchComponent,
+	GutenbergEditorPage,
+	LoginPage,
+	NewPostFlow,
+	setupHooks,
+	UserSignupPage,
+	SignupPickPlanPage,
+	AccountSettingsPage,
+	AccountClosedPage,
+	BrowserManager,
+	EmailClient,
+	NavbarComponent,
+	MeSidebarComponent,
+	BrowserHelper,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe(
+	DataHelper.createSuiteTitle( 'Signup: WordPress.com Free/Publish/Magic Link' ),
+	function () {
+		const inboxId = DataHelper.config.get( 'inviteInboxId' ) as string;
+		const username = `e2eflowtestingeditor${ DataHelper.getTimestamp() }`;
+		const email = DataHelper.getTestEmailAddress( {
+			inboxId: inboxId,
+			prefix: username,
+		} );
+		const signupPassword = DataHelper.config.get( 'passwordForNewTestSignUps' ) as string;
+		const blogName = DataHelper.getBlogName();
+
+		let page: Page;
+		let domainSearchComponent: DomainSearchComponent;
+		let gutenbergEditorPage: GutenbergEditorPage;
+
+		setupHooks( ( args ) => {
+			page = args.page;
+		} );
+
+		describe( 'Signup', function () {
+			it( 'Navigate to Signup page', async function () {
+				const loginPage = new LoginPage( page );
+				await loginPage.clickSignup();
+			} );
+
+			it( 'Sign up as new user', async function () {
+				const userSignupPage = new UserSignupPage( page );
+				await userSignupPage.signup( email, username, signupPassword );
+			} );
+
+			it( 'Select a free .wordpress.com domain', async function () {
+				domainSearchComponent = new DomainSearchComponent( page );
+				await domainSearchComponent.search( blogName );
+				await domainSearchComponent.selectDomain( '.wordpress.com' );
+			} );
+
+			it( 'Select WordPress.com Free plan', async function () {
+				const signupPickPlanPage = new SignupPickPlanPage( page );
+				await signupPickPlanPage.selectPlan( 'Free' );
+			} );
+		} );
+
+		describe( 'Interact with editor', function () {
+			it( 'Start a new post', async function () {
+				const newPostFlow = new NewPostFlow( page );
+				await newPostFlow.newPostFromNavbar();
+			} );
+
+			it( 'Return to Home dashboard', async function () {
+				// Temporary workaround due to https://github.com/Automattic/wp-calypso/issues/51162.
+				// Conditional can be removed once fixed.
+				if ( BrowserHelper.getTargetDeviceName() === 'mobile' ) {
+					await page.goBack();
+				} else {
+					await gutenbergEditorPage.openNavSidebar();
```

Comment:
Looks like you're missing the class instantiation here!
```suggestion
					gutenbergEditorPage = new GutenbergEditorPage( page );
					await gutenbergEditorPage.openNavSidebar();
```

Reply:
Hmm, rookie mistake, thanks for catching it. Fixed in 3c5d75c615396095f70524b6e99748c0ad17f321.